shared/teams/common/contacts-list.native.tsx

```diff
@@ -66,39 +68,54 @@ type Props = {
 
 type ContactRowProps = {
   disabled?: boolean
+  disabledTooltip?: string
   index: number
   item: Contact
   onSelect: Props['onSelect']
   selected: boolean
 }
-const ContactRow = React.memo(({item, disabled, index, onSelect, selected}: ContactRowProps) => {
-  const topText = item.name ?? item.valueFormatted ?? item.value
-  const bottomText = item.name ? item.valueFormatted ?? item.value : undefined
-  const onCheck = (check: boolean) => onSelect(item, check)
-  return (
-    <Kb.ListItem2
-      type="Small"
-      firstItem={index === 0}
-      body={
-        <Kb.Box2 direction="vertical" alignItems="flex-start">
-          <Kb.Text type="BodySemibold">{topText}</Kb.Text>
-          {bottomText && <Kb.Text type="BodySmall">{bottomText}</Kb.Text>}
-        </Kb.Box2>
-      }
-      onClick={disabled ? undefined : () => onCheck(!selected)}
-      action={
-        <Kb.CheckCircle checked={selected} onCheck={onCheck} style={styles.checkCircle} disabled={disabled} />
-      }
-      icon={
-        item.pictureUri ? (
-          <Kb.NativeImage style={styles.thumbnail} source={{uri: item.pictureUri}} />
-        ) : (
-          undefined
-        )
-      }
-    />
-  )
-})
+const ContactRow = React.memo(
+  ({item, disabled, disabledTooltip, index, onSelect, selected}: ContactRowProps) => {
+    const topText = item.name ?? item.valueFormatted ?? item.value
+    const bottomText = item.name ? item.valueFormatted ?? item.value : undefined
+    const onCheck = (check: boolean) => onSelect(item, check)
+    const listItem = (
+      <Kb.ListItem2
+        type="Small"
+        firstItem={index === 0}
+        body={
+          <Kb.Box2 direction="vertical" alignItems="flex-start">
+            <Kb.Text type="BodySemibold">{topText}</Kb.Text>
+            {bottomText && <Kb.Text type="BodySmall">{bottomText}</Kb.Text>}
+          </Kb.Box2>
+        }
+        onClick={disabled ? undefined : () => onCheck(!selected)}
+        action={
+          <Kb.CheckCircle
+            checked={selected}
+            onCheck={onCheck}
+            style={styles.checkCircle}
+            disabled={disabled}
+          />
+        }
+        icon={
+          item.pictureUri ? (
+            <Kb.NativeImage style={styles.thumbnail} source={{uri: item.pictureUri}} />
+          ) : (
+            undefined
```

Comment:
Wanna change this to a blank avatar to match the designs while you're here? I didn't know how to make a blank avatar when I made these.

Reply:
Sure